README.md

```diff
@@ -22,6 +20,13 @@ availability under heavy load or in the face of resource failure. **AbsaOSS/k3d-
 functionality**. For instance, the E2E use-case is [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) testing
  in [AbsaOSS/k8gb](https://github.com/AbsaOSS/k8gb). 
```

Comment:
https://github.com/k8gb-io/k8gb makes sense to switch to new location, if possible